pkg/ovs/openflow/ofctrl_bridge_test.go

```diff
@@ -160,3 +166,75 @@ func TestOFBridgePacketRcvd(t *testing.T) {
 		}
 	}
 }
+
+func TestOFMeterStats(t *testing.T) {
+	// Reset OVS metrics (Prometheus) and reinitialize them to test.
+	legacyregistry.Reset()
+	metrics.InitializeOVSMetrics()
+
+	b := NewOFBridge("test-br", GetMgmtAddress(ovsconfig.DefaultOVSRunDir, "test-br"))
+	sw := newFakeOFSwitch(b)
+	b.SetOFSwitch(sw)
+
+	mpMeterStatsReply := openflow15.NewMpReply(openflow15.MultipartType_MeterStats)
+	meterStats1 := openflow15.NewMeterStats(1)
+	stats1 := openflow15.NewMeterBandStats()
+	stats1.PacketBandCount = 100
+	stats1.ByteBandCount = 200
+	meterStats1.AddBandStats(*stats1)
```

Comment:
maybe create a closure to avoid repeating the same code 3 times:
```go
newMeterBandStats := func(meterID int, packets, bytes int) *MeterBandStats {

}
```

Reply:
Thanks for the suggestion, done.